### ISSUE TITLE
Nao_gazebo_plugin_H25.launch doesn't run correctly.

### ISSUE THREAD

**Issue by ban-masa** [ros-naoqi/nao_virtual#11]
Hi.
I can't launch nao_gazebo_plugin_H25.launch correctly.
It displays nao and the field, but it can't load required controllers and this error is output.
```
[WARN] [WallTime: 1447018352.432452] [0.000000] Controller Spawner couldn't find the expected controller_manager ROS interface.
```
In addition, it doesn't publish /tf.
However, when I checkout version 0.0.4, it runs correctly.

**Comment by keulYSMB** [comment]
It seems that the namespace has been removed in the last version. It will be fixed in nao_description soon, in the meantime you can just remove the namespace in [naoGazebo.xacro](https://github.com/ros-naoqi/nao_robot/blob/master/nao_description/urdf/naoV40_generated_urdf/naoGazebo.xacro#L76).

**Comment by vrabaud** [comment]
@keulYSMB , what do I need to do here ? Do I need to make a new release of something ?

**Comment by keulYSMB** [comment]
@vrabaud, I'm waiting for feedback from @nlyubova to either remove the
namespace from everywhere or getting back to the previous working version
with "nao_dcm" namespace everywhere.

**Comment by nlyubova** [comment]
Yes, it's better to remove. For me it works like it's now. What should I
check/ remove else exactly ?

**Comment by keulYSMB** [comment]
@nlyubova Can you help me for the testing part ? I don't have access to any
real NAO and I would like to be sure that by fixing the gazebo package we
dont endup breaking something running on the real robot.

**Comment by nlyubova** [comment]
Yes sure, What exactly should I test?

**Comment by zhudelong** [comment]
I got even more problems like:

[ INFO] [1448280160.397253365]: Camera Plugin (robotNamespace = /nao_robot), Info: Using the 'robotNamespace' param: '/nao_robot'
[ WARN] [1448280160.397327749]: gazebo_ros_camera_ simulation does not support non-zero distortion parameters right now, your simulation maybe wrong.
[ INFO] [1448280160.398905232]: Camera Plugin (robotNamespace = /nao_robot), Info: Using the 'robotNamespace' param: '/nao_robot'
[ WARN] [1448280160.398950005]: gazebo_ros_camera_ simulation does not support non-zero distortion parameters right now, your simulation maybe wrong.

I try to remove namespace but still not work. Anyone have an idea? thank you very much

**Comment by keulYSMB** [comment]
This is not an error that you should try to solve. We provide complete
models for the cameras, but this version of gazebo doesn't handle advances
camera models and thus does't simulate distortion.
This message is just a warnong to let you know that distortion won't be
simulated.

**Comment by keulYSMB** [comment]
@nlyubova can you test the branch fixControllers ?
Ideally I would like you to test it in a new workspace:
1) launch roslaunch nao_gazebo_plugin nao_gazebo_plugin_H25.launch
2) launch nao_dcm

I'm surprised the moveit package does not seem to be working anymore. any idea about that ? have you try it in a new setup since your changes ?

**Comment by keulYSMB** [comment]
BTW this should fix #12 once we figure out what's going on the moveit side

**Comment by keulYSMB** [comment]
I tried the new controllers on the gazebo package and they seem to be launched properly and to receive commands sent by rostopic pub. Could anyone test it on a real robot ?

**Comment by keulYSMB** [comment]
no need to rename controllers. bug fixed in ros-naoqi/nao_robot@f8eeae0
Tested on real and gazebo simulated robot
